ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/fragment/v1/fragment/README.md

```diff
@@ -0,0 +1,77 @@
+<!--
+Copyright 2023 Adobe
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+Adaptive Form Fragment (v1)
+====
+Adaptive Form Fragment component written in HTL.
+
+## Features
+Allow referencing a fragment resource.
+
+### Use Object
+The Fragment component uses the `com.adobe.cq.forms.core.components.models.form.Fragment` Sling Model for its Use-object.
+
+### Edit Dialog Properties
+The following properties are written to JCR for this Fragment component and are expected to be available as `Resource` properties:
+
+#### Fragment Properties
+1. `./jcr:title` - defines the label to use for this panel
+2. `./name` - defines the name of the panel, which will be submitted with the form data
+3. `./fragmentPath` - defines the path of the fragment resource
+4. `./bindref` - defines the data binding, and how data will be sent
+5. `./visible` - defines initial state of panel visibility
+6. `./enabled` - defines initial state of panel if its enabled or not
+7. `./tooltip` - defines tooltip on panel title
+8. `./description` - defines a help message that can be rendered in the field as a hint for the user
```

Comment:
when embedding or stitching in runtime, the above properties should be passed on in corresponding sling model and rendered accordingly.